server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java

```diff
@@ -79,7 +86,13 @@ void setUp() throws Exception {
         usersRepository.addUser(BOB.asString(), PASSWORD);
         MailboxSessionMapperFactory factory = null;
         userQuotaRootResolver = new DefaultUserQuotaRootResolver(factory);
-        UserQuotaService userQuotaService = new UserQuotaService(maxQuotaManager, userQuotaRootResolver);
+
+        currentQuotaManager = mock(CurrentQuotaManager.class);
+        Mockito.when(currentQuotaManager.getCurrentMessageCount(any())).thenReturn(QuotaCount.count(0));
+        Mockito.when(currentQuotaManager.getCurrentStorage(any())).thenReturn(QuotaSize.size(0));
```

Comment:
static import

Reply:
It's clashing with **RestAssured::when**